beetsplug/fetchart.py

```diff
@@ -210,6 +210,9 @@ def _candidate(self, **kwargs):
     def fetch_image(self, candidate, plugin):
         raise NotImplementedError()
 
+    def cleanup_tmp(self, candidate):
+        raise NotImplementedError()
```

Comment:
How about calling this method `cleanup` and making it `pass` in the base class? Then this will do nothing by default, and other sources can override it only if they need to do something on cleanup.

Reply:
That's fair.  Done.